CHANGELOG.md

```diff
@@ -78,6 +78,11 @@
 
 #### Fixed
 
+- When `Endpoints` could not be found for a `Service` to add them as targets of
+  a Kong `Upstream`, this was producing a log message at `error` level which was
+  inaccurate because this condition is often expected when `Pods` are being
+  provisioned. Those log entries now report at `info` level.
+  [#2820](https://github.com/Kong/kubernetes-ingress-controller/issues/2820)
```

Comment:
Do we have a guide/standard of whether to put links to issue or PR in `CHANGELOG`? I did not find that, so I think they are both OK.